src/d3a/models/area/event_dispatcher.py

```diff
@@ -101,7 +101,8 @@ def _should_dispatch_to_strategies_appliances(self, event_type):
             return self.area.events.is_connected and self.area.events.is_enabled
 
     def event_listener(self, event_type: Union[MarketEvent, AreaEvent], **kwargs):
-        if event_type is AreaEvent.TICK:
+        if event_type is AreaEvent.TICK and \
+                self._should_dispatch_to_strategies_appliances(event_type):
```

Comment:
Also something weird, why didnt this fail before, right?

Reply:
That surprised me too. I guess the reason for not finding it out until now is that this is useful only for the area events, and the integration tests for these were running only for one hour, therefore they were rendered useless. Now that the tests pass, the issue was revealed.